sympy/integrals/transforms.py

```diff
@@ -1137,6 +1190,607 @@ def sbs(expr):
         aux = _simplifyconds(aux, s, a)
     return _simplify(F.subs(s, s_), simplify), sbs(a), _canonical(sbs(aux))
 
+def _laplace_deep_collect(f, t):
+    func = f.func
+    args = [*f.args]
```

Comment:
I guess this is equivalent to `args = list(args)` but I haven't seen it written this way before.

Reply:
I will change it, then, I love the '*' unoack notation, but may have gone too far here ;)